src/app/services/mail/mail.types.ts

```diff
@@ -72,8 +72,11 @@ export type SubmissionToAdminHtmlData = {
 }
 
 export type AutoreplyHtmlData =
-  | ({ autoReplyBody: string[] } & AutoreplySummaryRenderData)
-  | { autoReplyBody: string[] }
+  | ({
+      responseId: string
+      autoReplyBody: string[]
+    } & AutoreplySummaryRenderData)
+  | { responseId: string; autoReplyBody: string[] }
```

Comment:
would it be better to extract `{ responseId: string; autoReplyBody: string[] }` into a separate type? this is the DRY principle

Reply:
also, let's try to always refer to the submission ID as `submissionId` in the code. I know we use the name `responseId` for user-facing communication, but let's keep it consistent in the code so devs always know what it refers to.